packages/fx-core/src/plugins/resource/frontend/configs.ts

```diff
@@ -41,7 +48,12 @@ export class FrontendConfig {
       throw new UnauthenticatedError();
     }
 
-    const appName = ctx.projectSettings!.appName;
+    let appName: string;
+    if (isArmSupportEnabled()) {
+      appName = getArmOutput(ctx, ArmOutput.frontendWebAppName) as string;
```

Comment:
according to the original code, the app name should be the Teams App's name, not the frontend hosting's resource name, we do not need to do extra things here.

Reply:
it is fixed. thanks.